cmd/dep/prune.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"flag"
+
+	"github.com/golang/dep"
+)
+
+const pruneShortHelp = `Prune was merged into ensure. Use ensure instead.`
```

Comment:
"Pruninng behavior has been merged into ensure. dep prune is now a no-op, and will be removed in the future."